src/components/Tutorial/TutorialUtils.js

```diff
@@ -2,26 +2,49 @@ import { store } from '../../store'
 import { TUTORIAL_CONTEXT, TUTORIAL_CONTEXT1_PARENT, TUTORIAL_CONTEXT2_PARENT } from '../../constants'
 import { getChildrenRanked } from '../../selectors'
 
+/** Kludgy workaround for undefined rootSubthoughts/rootChildren. */
+function getRootThing(rootSubthoughts, rootChildren) {
+  if (rootSubthoughts) {
+    // console.log('returning rootSubthoughts')
+    // if (rootChildren) {
+    //   console.log('  but have rootChildren also (WEIRD)')
+    // }
+    return rootSubthoughts
+  }
+  else {
+    if (rootChildren) {
+      // console.log('returning rootChildren')
+      return rootChildren
+    }
+    else {
+      // console.log('returning neither (FAIL)')
+      return undefined
+    }
+  }
+}
+
 /** Returns true if the first context thought has been created, e.g. /Home/To Do/x. */
-export const context1SubthoughtCreated = ({ rootSubthoughts, tutorialChoice }) => {
+export const context1SubthoughtCreated = ({ rootSubthoughts, rootChildren, tutorialChoice }) => {
```

Comment:
Why take both `rootSubthoughts` and `rootChildren`? They're the same thing, just accidentally split up due to a renaming mixup.